variables.tf

```diff
@@ -138,18 +143,6 @@ variable "spec" {
   default     = ""
 }
 
-variable "frontend_port" {
-  description = "Port used by the Server Load Balancer instance frontend."
-  type        = number
-  default     = 80
-}
-
-variable "protocol" {
-  description = "The protocol to listen on. Valid values are [http, https, tcp, udp]."
-  type        = string
-  default     = "http"
-}
-
 // DNS
 variable "bind_domain" {
   description = "Whether to bind domain."
```

Comment:
```
variable "create_slb" {
  description = "Whether to create slb."
  type        = bool
  default     = false
}
```
Whether to create a slb instance and attach the Ecs instance. If true, a new Slb instance and listener will be created and the Ecs instance will be attached to the Slb."